magento/magento2
# [MFTF] Coverage for the 30256 Unexpected unchecking of "Append Comments" check-box issue

### Description (*)
The test covers [the 30256 Unexpected unchecking of "Append Comments" check-box issue](https://github.com/magento/magento2/issues/30256)

### Related Pull Requests
https://github.com/magento/magento2/pull/31422

### Manual testing scenarios (*)
1. Start order creating in Admin
2. Add product
3. Provide a comment
4. Assert if the "Append Comment" checkbox is checked
5. Select payment method
6. Select shipping method
7. Assert if the "Append Comment" checkbox is checked
8. Place the order
9. On the Order Details page check if a customer is notified of the comment 



### Resolved issues:
1. [x] resolves magento/magento2#32381: WIP [MFTF] Coverage for the 30256 Unexpected unchecking of "Append Comments" check-box issue
